react/src/pages/RoutingListPage.tsx

```diff
@@ -259,6 +240,130 @@ const RoutingListPage: React.FC<RoutingListPageProps> = () => {
+      children: <Typography.Text copyable>{endpoint?.name}</Typography.Text>,
+    },
+    {
+      label: t('modelService.Status'),
+      children: <EndpointStatusTag endpointFrgmt={endpoint} />,
+    },
+    {
+      label: t('modelService.EndpointId'),
+      children: endpoint?.endpoint_id,
+    },
+    {
+      label: t('modelService.SessionOwner'),
+      children: <EndpointOwnerInfo endpointFrgmt={endpoint} />,
+    },
+    {
+      label: t('modelService.DesiredSessionCount'),
+      children: endpoint?.desired_session_count,
+    },
+    {
+      label: t('modelService.ServiceEndpoint'),
+      children: endpoint?.url ? (
+        <Typography.Text copyable>{endpoint?.url}</Typography.Text>
+      ) : (
+        <Typography.Text type="secondary">
+          {t('modelService.NoServiceEndpoint')}
+        </Typography.Text>
+      ),
+    },
+    {
+      label: t('modelService.OpenToPublic'),
+      children: endpoint?.open_to_public ? (
+        <CheckOutlined />
+      ) : (
+        <CloseOutlined />
+      ),
+    },
+    {
+      label: t('modelService.resources'),
+      children: (
+        <Flex direction="row" wrap="wrap" gap={'md'}>
+          <Tooltip title={t('session.ResourceGroup')}>
+            <Tag>{endpoint?.resource_group}</Tag>
+          </Tooltip>
+          {_.map(
+            JSON.parse(endpoint?.resource_slots || '{}'),
+            (value: string, type: ResourceTypeKey) => {
+              return (
+                <ResourceNumber
+                  key={type}
+                  type={type}
+                  value={value}
+                  opts={resource_opts}
+                />
+              );
+            },
+          )}
+        </Flex>
+      ),
+      span: {
+        xl: 2,
+      },
+    },
+    {
+      label: t('session.launcher.ModelStorage'),
+      children: (
+        <Suspense fallback={<Spin indicator={<LoadingOutlined spin />} />}>
+          <Flex direction="column" align="start">
+            <VFolderLazyView
+              uuid={endpoint?.model as string}
+              clickable={false}
+            />
+            {baiClient.supports('endpoint-extra-mounts') &&
+              endpoint?.model_mount_destination && (
+                <Flex direction="row" align="center" gap={'xxs'}>
+                  <ArrowRightOutlined type="secondary" />
+                  <Typography.Text type="secondary">
+                    {endpoint?.model_mount_destination}
+                  </Typography.Text>
+                </Flex>
+              )}
+          </Flex>
+        </Suspense>
+      ),
+    },
+  ];
+
+  if (baiClient.supports('endpoint-extra-mounts')) {
+    items.push({
+      label: t('modelService.AdditionalMounts'),
+      children:
+        (endpoint?.extra_mounts?.length as number) > 0 ? (
+          _.map(endpoint?.extra_mounts, (vfolder) => (
+            <VFolderLazyView
+              uuid={vfolder?.row_id as string}
+              clickable={false}
+            />
```

Comment:
Oh! This part should be reverted to using `vfolder.name` instead of `VFolderLazyView`

Reply:
Bulls eye! I just fixed this, too.